README.md

```diff
@@ -1,29 +1,55 @@
-# Morello-API
-The interface for executing binaries on morello board and self that returns output which can be processed by the client.
+# morello-api
 
+## Description
```

Comment:
I'm not sure why but if you go "view file" this isn't rendering on github correctly

Reply:
good spot! There was something wrong with the whitespace character?